annotator-core/src/test/java/edu/ucr/cs/riple/core/tools/CoreTestHelper.java

```diff
@@ -312,6 +316,24 @@ public void makeAnnotatorConfigFile(Path configPath) {
     builder.write(configPath);
   }
 
+  /**
+   * Returns true, if the environment variable is not set, or negate of the variable if set. The
+   * reason this method is acting in reverse, is because the default configuration when running the
+   * tests is to activate both cache and parallel processing in an arbitrary environment where these
+   * environment variables are not set. Variable names are `ANNOTATOR_TEST_DISABLE_CACHING` and
+   * `ANNOTATOR_TEST_DISABLE_PARALLEL_PROCESSING` which both disables their corresponding features.
+   *
+   * @param environmentVariableName environment variable name.
+   * @return true if the environment variable is not set, or negate of the variable if set.
+   */
+  private boolean getModeFromEnvironment(String environmentVariableName) {
+    String value = System.getenv(environmentVariableName);
+    if (value == null || value.isEmpty()) {
+      return true;
+    }
+    return !Boolean.parseBoolean(value);
```

Comment:
Can we move the negation the callsites? I am just thinking if we will ever have other environment variables here, we might not _always_ keep the convention that all are `ANNOTATOR_TEST_DISABLE_{FEATURE}` flags. 

And either way, this seems strictly more clear:

```
    // Both of these are enabled unless specified otherwise in env vars:
    builder.useCacheImpact = !getModeFromEnvironment("ANNOTATOR_TEST_DISABLE_CACHING");
    builder.useParallelProcessor =
        !getModeFromEnvironment("ANNOTATOR_TEST_DISABLE_PARALLEL_PROCESSING");
```

And we get to simplify the above comment.

Reply:
@lazaroclapp The reason that I did the negation inside the function was to get rid of the warnings from IDE that the return value is always inverted 🙂! But you are absolutely right, and for this case it does not make sense to write this method in this way especially about the convention `ANNOTATOR_TEST_DISABLE_{FEATURE} `. Moved the negation to call sites c3c8a2f8105bffe29dac8045a73c33fde0e835b7